cli/info/info.go

```diff
@@ -44,9 +47,35 @@ func infoMain(args []string, c cli.Config) (err error) {
 	if err != nil {
 		return err
 	}
+
+	_, err = helpers.ParseCertificatePEM(certPem)
+	if err != nil {
+		return errors.Wrap(errors.CertificateError, errors.DecodeFailed, err)
+	}
+
+	s, err := sign.SignerFromConfig(c)
```

Comment:
@dr107 There's a bit more plumbing needed to get the full feature.

There are two cases: 1) when -remote is not empty 2) when -remote is empty and -config is not
  
In 1) use the -remote and the api/client package if the -remote flag is there. The api/client package should be updated to return the Usage and ExpiryString that the info API returns. Right now it only returns the certificate.

If the -remote flag is not there, then it's 2). In this case continue to use the config package to get a signer.Signer using sign.SignerFromConfig(c). Update the code in the various signers so that s.Info() returns Usage and ExpiryString along with Certificate, and use those to construct the InfoResp.